### PR TITLE
Add user.reset_password_to_default action + test.

### PR DESCRIPTION
**Pull request by reiterl**
Assume that in default_password is the hash value which can be copied into password.
Add a short test.

### CODE REVIEW

File: openslides_backend/action/user/reset_password_to_default.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Gets the default_password and reset password.
+        """
+        user = self.database.get(
+            FullQualifiedId(self.model.collection, instance["id"]), ["default_password"]
+        )
+        default_password = user.get("default_password")
+        instance["password"] = default_password
```
Comment (jsangmeister) [inline openslides_backend/action/user/reset_password_to_default.py:27]: `default_password` password contains the clear text pw. Please hash it first

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/user/reset_password_to_default.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/user/reset_password_to_default.py:27]: Done.

**Review by jsangmeister — APPROVED** [review]
Please squash and then it can be merged

**Comment by jsangmeister** [comment]
Done.

**Pull request merged**